### PR TITLE
ignoring system user from /i responders

### PR DESCRIPTION
**Pull request by jmervine**
addressing #211

### CODE REVIEW

File: spec/lib/chat_ops/message_helpers_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+RSpec.describe ChatOps::MessageHelpers do
+  describe ".return_responders" do
+    let(:userR) { create(:retrodot_user) }
+
+    let(:incident) do
+      i = create(:incident_with_responder)
+      i.responders << userR
+      i.save!
+      i
+    end
```
Comment (joshuatobin) [inline spec/lib/chat_ops/message_helpers_spec.rb]: I believe you can do stuff like this in the factory with after create. This way is fine but I thought i'd mention.

**Comment by joshuatobin** [comment]
👍

File: spec/lib/chat_ops/message_helpers_spec.rb (hunk 1 above)
Comment (jmervine) [inline spec/lib/chat_ops/message_helpers_spec.rb]: Maybe but I need you to teach me that crazy Gandalf style magic. I couldn't figure it out. 😂

Comment (joshuatobin) [inline spec/lib/chat_ops/message_helpers_spec.rb]: Yeah its weird. :shipit: for now

**Pull request merged**